opentelemetry/proto/collector/entities/v1development/entities_service.proto

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opentelemetry.proto.collector.entities.v1development;
+
+import "opentelemetry/proto/entities/v1development/entities.proto";
+
+option csharp_namespace = "OpenTelemetry.Proto.Collector.Entities.V1Development";
+option java_multiple_files = true;
+option java_package = "io.opentelemetry.proto.collector.entities.v1development";
+option java_outer_classname = "EntitiesServiceProto";
+option go_package = "go.opentelemetry.io/proto/otlp/collector/entities/v1development";
+
+// Service that can be used to push entities between one Application instrumented with
+// OpenTelemetry and a collector, or between a collector and a central collector (in this
```

Comment:
wouldn't that imply that collectors loose their stateless scalability property? that change would imply that collectors need to receive all entity event to deliver entity-based enrichment